zapetw/write_syncer.go

```diff
@@ -0,0 +1,76 @@
+//go:build windows
+// +build windows
```

Comment:
If you name the file *_windows.go, this build tag should be implied by the toolchain.